opensrp-chw-core/src/main/java/org/smartregister/chw/core/activity/CoreAllClientsRegisterActivity.java

```diff
@@ -63,4 +85,37 @@ public void switchToBaseFragment() {
     public void startRegistration() {
         //implement
```

Comment:
Will this be implemented in this PR? If not could we create a ticket to track this?